app/account/account.html

```diff
@@ -1,70 +1,39 @@
 <ion-view title="Account" id="tab-account">
+
+
   <ion-content padding="true" class="has-header">
 
-    <form name="AccountCtrl.accountForm" id="AccountCtrl.accountForm" class="list">
-      <div>
-        <label class="item item-input">
-          <span class="input-label">LastName:</span>
-          <p data-ng-bind="AccountCtrl.account.lastName"></p>
-        </label>
-      </div>
-      <div>
-        <label class="item item-input">
-          <span class="input-label">First Name:</span>
-          <p data-ng-bind="AccountCtrl.account.firstName"></p>
-        </label>
-      </div>
-      <div>
-        <label class="item item-input">
-          <span class="input-label">Email:</span>
-          <p data-ng-bind="AccountCtrl.account.email"></p>
-        </label>
-      </div>
-      <div>
-        <label class="item item-input">
-          <span class="input-label">Phone:</span>
-          <p data-ng-bind="AccountCtrl.account.phone"></p>
-        </label>
-      </div>
-      <div>
-        <label class="item item-input">
-          <span class="input-label">Birth Date:</span>
-          <p data-ng-bind="AccountCtrl.account.birthdate"></p>
-        </label>
-      </div>
-      <div>
-        <h2 class="text-center">Address</h2>
-      </div>
-      <div>
-        <label class="item item-input">
-          <span class="input-label">N°</span>
-          <p required type="text" placeholder="N°" data-ng-bind="AccountCtrl.account.address.number">
-        </label>
-      </div>
-      <div>
-        <label class="item item-input">
-          <span class="input-label">Street</span>
-          <p required type="text" placeholder="street" data-ng-bind="AccountCtrl.account.address.street">
-        </label>
-      </div>
-      <div>
-        <label class="item item-input">
-          <span class="input-label">Postal Code</span>
-          <p required type="number" placeholder="zip" data-ng-bind="AccountCtrl.account.address.zip">
-        </label>
-      </div>
-      <div>
-        <label class="item item-input">
-          <span class="input-label">City</span>
-          <p required type="text" placeholder="city" data-ng-bind="AccountCtrl.account.address.city">
-        </label>
-      </div>
-      <div>
-        <label class="item item-input">
-          <span class="input-label">Country</span>
-          <p required type="text" placeholder="country" data-ng-bind="AccountCtrl.account.address.country">
-        </label>
-      </div>
-    </form>
+
+
+    <ion-list inset>
+      <ion-item class="item" *ngFor="let item of items" (click)="itemSelected(item)">
```

Comment:
To delete, quand tu regardes la doc ionic là tu as regardé la doc ionic 2 et pas ionic 1 Attention http://ionicframework.com/docs/